js/compiler/compiler.go

```diff
@@ -1,94 +1,29 @@
-			// es2015 https://github.com/babel/babel/blob/v6.26.0/packages/babel-preset-es2015/src/index.js
-			// in Sobek
-			// []interface{}{"transform-es2015-template-literals", map[string]interface{}{"loose": false, "spec": false}},
-			// "transform-es2015-literals", // in Sobek
-			// "transform-es2015-function-name", // in Sobek
-			// []interface{}{"transform-es2015-arrow-functions", map[string]interface{}{"spec": false}}, // in Sobek
-			// "transform-es2015-block-scoped-functions", // in Sobek
-			// []interface{}{"transform-es2015-classes", map[string]interface{}{"loose": false}}, // in Sobek
-			// "transform-es2015-object-super", // in Sobek
-			// "transform-es2015-shorthand-properties", // in Sobek
-			// "transform-es2015-duplicate-keys", // in Sobek
-			// []interface{}{"transform-es2015-computed-properties", map[string]interface{}{"loose": false}}, // in Sobek
-			// "transform-es2015-for-of", // in Sobek
-			// "transform-es2015-sticky-regex", // in Sobek
-			// "transform-es2015-unicode-regex", // in Sobek
-			// "check-es2015-constants", // in Sobek
-			// []interface{}{"transform-es2015-spread", map[string]interface{}{"loose": false}}, // in Sobek
-			// "transform-es2015-parameters", // in Sobek
-			// []interface{}{"transform-es2015-destructuring", map[string]interface{}{"loose": false}}, // in Sobek
-			// "transform-es2015-block-scoping", // in Sobek
-			// "transform-es2015-typeof-symbol", // in Sobek
-			// all the other module plugins are just dropped
-			[]interface{}{"transform-es2015-modules-commonjs", map[string]interface{}{"loose": false}},
-			// "transform-regenerator", // Doesn't really work unless regeneratorRuntime is also added
-
-			// es2016 https://github.com/babel/babel/blob/v6.26.0/packages/babel-preset-es2016/src/index.js
-			// "transform-exponentiation-operator",
-
-			// es2017 https://github.com/babel/babel/blob/v6.26.0/packages/babel-preset-es2017/src/index.js
-			// "syntax-trailing-function-commas", // in Sobek
-			// "transform-async-to-generator", // Doesn't really work unless regeneratorRuntime is also added
-		},
-		"ast":           false,
-		"sourceMaps":    false,
-		"babelrc":       false,
-		"compact":       false,
-		"retainLines":   true,
-		"highlightCode": false,
-	}
-
-	maxSrcLenForBabelSourceMap     = 250 * 1024 //nolint:gochecknoglobals
-	maxSrcLenForBabelSourceMapOnce sync.Once    //nolint:gochecknoglobals
-
-	onceBabelCode      sync.Once      //nolint:gochecknoglobals
-	globalBabelCode    *sobek.Program //nolint:gochecknoglobals
-	errGlobalBabelCode error          //nolint:gochecknoglobals
-	onceBabel          sync.Once      //nolint:gochecknoglobals
-	globalBabel        *babel         //nolint:gochecknoglobals
-)
-
-const (
-	maxSrcLenForBabelSourceMapVarName = "K6_DEBUG_SOURCEMAP_FILESIZE_LIMIT"
-	sourceMapURLFromBabel             = "k6://internal-should-not-leak/file.map"
-)
+// A Compiler compiles JavaScript source code (ES5.1 or ES6) into a sobek.Program
```

Comment:
```suggestion
```

Looks like this comment is duplicated from the one below.

Reply:
I already have a branch that goes and fixes this and other comments, I would prefer to not block this PR on this.

There is going to quite a lot of cleanup either way.